addons/html_builder/__manifest__.py

```diff
@@ -25,6 +25,7 @@
         'html_builder.assets': [
             ('include', 'web._assets_helpers'),
             ('after', 'web/static/src/scss/primary_variables.scss', 'html_builder/static/src/builder.variables.scss'),
+            'html_builder/static/src/**/*.variables.scss',
```

Comment:
Same remark, could be in variable sub-assets instead of duplicated in each "relevant" assets